sources/osgShadow/ShadowReceiveAttribute.js

```diff
@@ -150,6 +158,14 @@ utils.createPrototypeStateAttribute(
             if (this.getAtlas()) defines.push('#define _ATLAS_SHADOW');
             if (this.getNormalBias()) defines.push('#define _NORMAL_OFFSET');
 
+            switch (this.getJitterOffset()) {
```

Comment:
switch seems overkill here
`if(this.getJitterOffset()!=='none') defines.push('#define _JITTER_OFFSET');`

(and it's consistent at least)